intranet/apps/polls/models.py

```diff
@@ -94,6 +94,30 @@ def get_users_voted(self):
                 users = list(q.get_users_voted())
         return users
 
+    def get_num_eligible_voters(self):
+        groups = get_user_model().objects.none()
+        if self.groups.exists():
+            q_objects = Q(name__in=[])
+            for g in self.groups.all():
+                q_objects.add(Q(name=g), Q.OR)
+            groups = DjangoGroup.objects.filter(q_objects)
+            users = set()
+            for g in groups:
+                users = users | set(g.user_set.all())
+            return len(users)
+        else:
+            return get_user_model().objects.count() - 1
+
+    def get_percentage_voted(self, voted, able):
+        percentage = voted / able
+        return "{:.1%}".format(percentage)
+
+    def get_voted_string(self):
+        users_voted = len(self.get_users_voted())
+        users_able = self.get_num_eligible_voters()
```

Comment:
The edge case of eligible voters being 0 should be handled to prevent a divide-by-zero error in `get_percentage_voted`. Although it might seem like an implausible scenario, there is the possibility of a poll creator creating a group authorized to vote in a poll that eventually is cleared to zero.